scripts/deploy.sh

```diff
@@ -168,11 +186,13 @@ function deploy_control_panel() {
 
   echo "Building the control_panel wasm..."
```

Comment:
```suggestion
  echo "Preparing the control_panel wasm..."
```
just to be consistent with the above